app/javascript/src/components/Contacts/Table/index.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+
+// Hit a width issue with Neetoui table
+// Using Antd Table, as NeetoUI table is using Antdesign Table inside
+
+import { MenuHorizontal } from "@bigbinary/neeto-icons";
+import { Avatar, Dropdown } from "@bigbinary/neetoui/v2";
+import { Table } from "antd";
```

Comment:
Using Ant Design table, Had a width issue using the table in NeetoUI. on giving w-full, Table was causing overflow of entire parent div. Used Ant as NeetoUI Table uses this internally.

Reply:
Please add a Loom video for the same. We should be using neetoUI `Table` over `antD`.

Reply:
[Here](https://www.loom.com/share/04ed8bf2c1f64473aba7381a3fbf45a8) is a loom video of the issue.

Reply:
We are using a ref to the parentContainer to calculate the width and height of neetoUI table. Please refer https://github.com/bigbinary/neeto-ui/blob/689b1789c4bd4da7bcbe629866cc8fdd13a8547e/example/src/Pages/Table.js#L188 for more details.

Reply:
Added parent ref, the table still has the same issue. Also while using the Neeto UI table I am having minor frame drops in Sidemenu/sidebar opening animations.